src/app/api/v1/fxa-rp-events/route.ts

```diff
@@ -286,8 +286,31 @@ export async function POST(request: NextRequest) {
             }
 
             // activate and opt out profiles
-            await activateProfile(oneRepProfileId);
-            await optoutProfile(oneRepProfileId);
+            try {
```

Comment:
Not in this PR, but I'm wondering why we have two info logging for this case `FXA_SUBSCRIPTION_CHANGE_EVENT`, L244